GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -75,31 +75,31 @@
 
     private static final String TAG = Log.tag(GVRViewManager.class);
 
-    private final Queue<Runnable> mRunnables = new LinkedBlockingQueue<Runnable>();
+    protected final Queue<Runnable> mRunnables = new LinkedBlockingQueue<Runnable>();
```

Comment:
Did all these need to be made `protected`?

(`protected` in Java means more than it does in sane languages: visible to descendants and all other code in package.)